static/app/components/dropdownMenuControl.tsx

```diff
@@ -58,28 +70,19 @@ interface DropdownMenuControlProps
    * Pass class name to the outer wrap
    */
   className?: string;
-  /**
-   * If this is a submenu, it will in some cases need to close itself (e.g.
-   * when the user presses the arrow left key)
-   */
-  closeCurrentSubmenu?: () => void;
-  /**
-   * If this is a submenu, it will in some cases need to close the root menu
-   * (e.g. when a submenu item is clicked).
-   */
-  closeRootMenu?: () => void;
   /**
    * Whether the trigger is disabled.
    */
   isDisabled?: boolean;
-  /**
-   * Whether this is a submenu.
-   */
-  isSubmenu?: boolean;
   /**
    * Title for the current menu.
    */
   menuTitle?: string;
+  /**
+   * Whether the menu should always be wider than the trigger. If true (default), then
+   * the menu will have a min width equal to the trigger's width.
+   */
+  menuWiderThanTrigger?: boolean;
```

Comment:
This prop will be false for submenus. Not sure if this is the best name — open to suggestions!